### PR TITLE
Added our team page in menu and a team folder

### PR DESCRIPTION
**Pull request by rava-dosa**

### CODE REVIEW

File: index.html~ [hunk 1]
```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
```
Comment (pranitbauva1997) [inline index.html~]: Please remove redundant files like this.

File: team/index.html~ [hunk 2]
```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
```
Comment (pranitbauva1997) [inline team/index.html~]: Please remove this redundant file as well

**Review by pranitbauva1997 — CHANGES_REQUESTED** [review]
I have not checked other parts. Let's see the deployed version first.

**Comment by pranitbauva1997** [comment]
@rava-dosa Please squash the changes using a CLI. Adding redundancies in commits is also not good.

**Comment by rava-dosa** [comment]
@pranitbauva1997 done

**Review by pranitbauva1997 — APPROVED** [review]

**Review by americast — COMMENTED** [review]
Waiting for a fix for #20

**Review by americast — COMMENTED** [review]
Merging it now to continue further work...

**Pull request merged**